src/module/combat/combat.js

```diff
@@ -552,6 +552,159 @@ export class CombatSFRPG extends Combat {
         return this.getIndexOfFirstUndefeatedCombatant() === null;
     }
 
+    getEncounterInfo() {
+        if (!this.flags.sfrpg) {
+            this.flags.sfrpg = [];
```

Comment:
Just spotted this too; `this.flags.sfrpg` should be an object

Reply:
Fixed